### PR TITLE
assign patient registration date to date_opened instead of date_reported

### PR DESCRIPTION
**Pull request by nickpell**
@czue @proteusvacuum

### CODE REVIEW

File: custom/enikshay/nikshay_datamigration/factory.py [hunk 1]
```diff
@@ -181,9 +181,9 @@ def get_episode_case_structure(self, occurrence_structure):
         kwargs = {
             'attrs': {
                 'case_type': EPISODE_CASE_TYPE,
+                'date_opened': self.patient_detail.pregdate1,
```
Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: yeah, obviously `date_opened`, sets the `opened_on` property 😕

**Review by proteusvacuum — APPROVED** [review]

**Review by czue — APPROVED** [review]
nice

File: custom/enikshay/nikshay_datamigration/factory.py (hunk 1 above)
Comment (czue) [inline custom/enikshay/nikshay_datamigration/factory.py]: someone hasn't been reading their casexml 2.0 specifications closely!

Comment (proteusvacuum) [inline custom/enikshay/nikshay_datamigration/factory.py]: I do believe reading that particular line of the casexml spec too closely resulted in me bricking a bunch of phones.

**Pull request merged**